Makefile

```
@@ -80,7 +80,7 @@ bin/kubebuilder: bin/kubebuilder_${KUBEBUILDER_VERSION}
 test: generate fmt vet manifests bin/kubebuilder
 	@which kubebuilder
 	@which etcd
-	kubebuilder version
+	bin/kubebuilder version
```

this was intentional. given the PATH env variable is set to the local bin directory this should work without specifing `bin/` here
```suggestion
	kubebuilder version
```

---

It doesn't work for me